sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1379,6 +1384,97 @@ static void _process_events_await_msg(gnrc_netif_t *netif, msg_t *msg)
     }
 }
 
+#ifdef MODULE_GNRC_NETIF_PKTQ
+static void _sched_dequeue(gnrc_netif_t *netif)
+{
+    netif->dequeue_msg.type = GNRC_NETIF_PKTQ_DEQUEUE_MSG;
+    unsigned state = irq_disable();
```

Comment:
Why do we need to disable IRQs here?

Reply:
Sorry, should have been clear that changes are still incoming. I moved this function to `gnrc_netif_pktq` itself now and there is an explaining comment.